### ISSUE TITLE
help - validate SCD dont change the score (finetuned for 100 epochs im my own data)

### ISSUE THREAD

**Issue by Gabrielk99** [pyannote/pyannote-audio#700]
Hello, sorry, but i aply the process of finetuning two times in the scd model, firts, for 5 epochs, and validate, this return a score of 98.6%, and the result is very bad im my data, in second time, I was used 100 epochs, and this generate a loss of 1.74x10⁻3, lass than the first process of finetuning, but in process validate, the score is the same?? in all epochs, i cant understand what is wrong with this, please u can help me?

**Comment by Gabrielk99** [comment]
@hbredin, please check if i use the correct comand line to validate process
 pyannote-audio scd  validate --pretrained="tutorials/finetune/SCD_FINETUNED_VALIDATED/train/train/VALE.SpeakerDiarization.Cambio.train/weights/0100.pt"  --subset=development --from=10  --to=100 --every=1 "tutorials/finetune/SCD_FINETUNED_VALIDATED/train/train/validate/VALE.SpeakerDiarization.Cambio.development" VALE.SpeakerDiarization.Cambio

this return Waiting for first epoch to complete... (is normal?)

**Comment by Gabrielk99** [comment]
in first time i used pyannote-audio scd validate --from= 10 --to=50 --every=10 {$TRN_DIR} procotocol

TRN_DIR is the directory of finetuned scd model, and dont change the score.
